wolfcrypt/src/asn.c

```diff
@@ -16991,6 +16984,17 @@ void InitOcspResponse(OcspResponse* resp, CertStatus* status,
     resp->status         = status;
     resp->source         = source;
     resp->maxIdx         = inSz;
+    resp->heap           = heap;
+}
+
+void FreeOcspResponse(OcspResponse* resp)
+{
```

Comment:
check resp != NULL

Reply:
This is an internal function and the other OCSP API's also do not check args. Seems like a good future cleanup operation...